_delphi_utils_python/Makefile

```
@@ -0,0 +1,22 @@
+.PHONY = venv, lint, test, clean
+
+dir = $(shell find ./delphi_* -name __init__.py |  sed 's/\/__init__.py//g;s/\.\///g')
```

This is 100% picky bash style nonsense, but slight preference for grep over sed, especially if the sed is going to require escapes:

```suggestion
dir = $(shell find ./delphi_* -name __init__.py | grep -o 'delphi_[_[:alnum:]]*')
```

---

:+1: